src/testing/lib/reportClass.php

```diff
@@ -311,18 +311,18 @@ function globdata($results, $moData)
 			}
 
 			$suiteName = $this->parseSuiteName($moData[$suite]);
-			array_push($results, $suiteName);
+			$results[] = $suiteName;
 			//print "parsed suite name:$suiteName\n";
 
 			$pfe_results = $this->parseResults($moData[$suite +1]);
-			$pfe = explode(':', $pfe_results);
-			array_push($results, $pfe[0]);
-			array_push($results, $pfe[1]);
-			array_push($results, $pfe[2]);
+			$pfe = split(':', $pfe_results);
```

Comment:
`split()` was deprecated as per #2606
```suggestion
			$pfe = explode(':', $pfe_results);
```